api/src/services/agencies/agencies.ts

```diff
@@ -35,6 +36,40 @@ export const deleteAgency: MutationResolvers['deleteAgency'] = ({ id }) => {
   })
 }
 
+export const agenciesAvailableForUpload = async () => {
```

Comment:
Named this resolver functionally, although it might be a good idea to investigate whether we can merge it with the existing non-role-specific `agenciesByOrganization` resolver.